ossindex/ossindex.go

```diff
@@ -139,7 +141,7 @@ func AuditPackages(purls []string) ([]types.Coordinate, error) {
 
 			defer func() {
 				if err := resp.Body.Close(); err != nil {
-					log.Printf("error closing response body: %s\n", err)
+					appLog.Debugf("error closing response body: %s\n", err)
```

Comment:
Did you not do appLog.Error here b/c we did not Exit??

Reply:
Oversight on my part! I can switch to error easily